scripts/skills/actives/rf_deep_impact_skill.nut

```diff
@@ -12,138 +13,97 @@ this.rf_deep_impact_skill <- ::inherit("scripts/skills/skill", {
 		this.m.Icon = "skills/rf_deep_impact_skill.png";
 		this.m.IconDisabled = "skills/rf_deep_impact_skill_sw.png";
 		this.m.Overlay = "rf_deep_impact_skill";
-		this.m.Type = ::Const.SkillType.Active;
-		this.m.Order = ::Const.SkillOrder.OffensiveTargeted;
-		this.m.IsSerialized = false;
-		this.m.IsActive = true;
```

Comment:
This skill as I understand it should remaing an active skill, so this line should stay